tests/authTests/DeleteSessionTests.py

```diff
@@ -0,0 +1,61 @@
+import json
+from unittest import main
+import requests
+
+from TestConfiguration import TestConfiguration
+
+class DeleteSessionTests(TestConfiguration):
+
+   def setUp(self):      
+      registerUrl = self.baseUrl + '/users'
+      registerBody = {
+         'firstName' : 'Dave',
+         'lastName' : 'Janzen',
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      registerResponse = requests.post(url=registerUrl, data=json.dumps(registerBody), headers=self.headers)
+
+      loginUrl = self.baseUrl + '/users/session'
+      loginBody = {
+         'email' : 'dave.janzen18@gmail.com',
+         'password' : 'Stockd2g'
+      }
+      loginResponse = requests.post(url=loginUrl, data=json.dumps(loginBody), headers=self.headers)
+      loginResponseData = loginResponse.json()
+      self.userId = loginResponseData['userId']
+      self.token = loginResponseData['token']
+      
+      self.url = self.baseUrl + '/users/' + str(self.userId) + '/session'
+      self.headers['Authorization'] = 'token ' + self.token
+
+
+   def test_logout_user(self):
+      response = requests.delete(url=self.url, headers=self.headers)     
+      try:
```

Comment:
I did it that way for consistency with all the other tests. It's almost a placeholder because I wanted a better output mechanism but wasn't able to get exactly what I wanted. We can discuss this for a hot second too.